### ISSUE TITLE
Single-file target path fails if it contains non-existing directories

### ISSUE THREAD

**Issue by reitzig** [reitzig/container-do#16]
```toml
# ContainerDo.toml
[container]
image = "ubuntu"
mounts = []

[[copy.setup]]
files = ["foo"]
to = "some/dir/foobar"
```

**Expected:** Container starts and file `some/dir/foobar` exists in container working directory.

**Actual:** Error because `some/dir` does not exist. (even after fixing #14 )

**Comment by reitzig** [comment]
**Analysis:** We only `mkdir -p` if  target` is explicitly labelled as directory (`/` suffix) or if there are multiple files to copy (which means the target _has_ to be a directory). 
We missed that in the other cases, we need to `mkdir -p` the `dirname` of the target.

**Comment by reitzig** [comment]
Fixed. Will be released with 1.2.1.
